selectors/lhs.ts

```diff
@@ -1,8 +1,32 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import {createSelector} from 'reselect';
 import {GlobalState} from 'types/store';
+import {makeGetDraftsCount} from 'selectors/drafts';
+
+import {getLhsStaticItems} from './views/lhs';
 
 export function getIsLhsOpen(state: GlobalState): boolean {
     return state.views.lhs.isOpen;
 }
+
+export const getVisibleLhsStaticItems = createSelector(
+    'getVisibleSidebarStaticItems',
+    (state: GlobalState) => getLhsStaticItems(state),
+    (state: GlobalState) => {
+        const getDraftsCount = makeGetDraftsCount();
```

Comment:
I'm not actually sure why `getDraftsCount` uses a selector, but for this to memoize properly, you need to create one copy of it outside of this scope (like just above the definition of `getVisibleLhsStaticItems`) and then you can use it here like any other selector.

These input selectors get ran every time the selector is called, and because we're creating a new copy of `getDraftsCount` each time, that means that no memoization is happening for `getVisibleLhsStaticItems` and the component always re-renders

Reply:
You're right, I think one of my tests was falling with having that outside and forgot it there, but I'll tinker with it so it can work properly.

Reply:
Let me know if you're running into issues with getting that working. Selectors can be a pain sometimes, so I'd be glad to help